src/components/TradeWidget/Price.tsx

```diff
@@ -89,6 +102,10 @@ const PriceInputBox = styled.div`
       color: #218dff;
     }
 
+    &:focus::placeholder {
+      color: transparent;
```

Comment:
This is cool but htere are no placeholders in PriceInputBox > input
Planning to add some?

Reply:
This was an attempt to implement a CSS only solution where the placeholder (usually a '0') is visually removed once you focus on it.

But indeed, there didn't seem to exist a placeholder when I fully tested it, but just left the styles in place.

Yes, if we can implement the default 0's as placeholders that would be cool.

Reply:
This is late, but btw `visibility: hidden` would also work